### PR TITLE
Update kr.js

### PR DESCRIPTION
**Pull request by YuniQ152**
<3

### CODE REVIEW

**Review by apinklover — CHANGES_REQUESTED** [review]
Line 174: How about "KDR 표시"
Line 244: How about leaving out the parenthetical extra information?
Line 249: Pretty sure it's "게임 제목 숨기기"
Line 261: Change it to "게임패드 비활성화" for translation consistency
Line 339: Change "출발" to "시작" ?

File: kr.js [hunk 1]
```diff
@@ -67,19 +67,19 @@ player.disconnect 연결 끊기
...
 leaderboard.empty 빈 로비
 popup.checkpoint 체크포인트!
-app.play 플레이하려면 클릭하세요
-app.spectate CLICK TO SPECTATE
-app.play.mobile TAP TO PLAY
-app.spectate.mobile TAP TO SPECTATE
+app.play 플레이하려면 클릭하세요.
+app.spectate 관전하려면 클릭하세요.
+app.play.mobile 플레이하려면 누르세요.
+app.spectate.mobile 관전하려면 누르세요.
```
Comment (mty22) [inline kr.js:73]: can you please remove the full stop (.) at the end on these lines please!

File: kr.js [hunk 2]
```diff
@@ -67,19 +67,19 @@ player.disconnect 연결 끊기
...
 app.play.controller 플레이하려면 START를 누르세요.
-app.spectate.controller START to SPECTATE
+app.spectate.controller 관전하려면 START를 누르세요.
```
Comment (mty22) [inline kr.js:75]: same here, can you please remove the full stop (.) :)

**Review by mty22 — CHANGES_REQUESTED** [review]
Just need to change a few things. Have a look at @apinklover 's comment too please!

Also with your next commit(s), please add 'Resolves: #9' to the commit message. :)

**Comment by dojinshi** [comment]
Changes looks awesome! Great work guys! :)

**Review by mty22 — APPROVED** [review]

**Comment by mty22** [comment]
@apinklover recheck please

**Comment by apinklover** [comment]
Looks good. I approve

**Comment by mty22** [comment]
Cheers!

**Pull request merged**